src/common/sizer.cpp

```diff
@@ -676,15 +677,45 @@ wxSizer::~wxSizer()
 
 wxSizerItem* wxSizer::DoInsert( size_t index, wxSizerItem *item )
 {
-    m_children.Insert( index, item );
+    // The helper class that solves two problems when
+    // wxWindowBase::SetContainingSizer() throws:
+    // 1. Avoid the memory leaking using the scoped pointer to the sizer item.
```

Comment:
```suggestion
    // 1. Avoid leaking memory using the scoped pointer to the sizer item.
```
(or "Avoid the memory leaks")